### ISSUE TITLE
Creating a markdown output would be a great feature

### ISSUE THREAD

**Issue by akshaydeo** [go-shiori/go-readability#11]
Instead of storing a txt file, if the tool could generate an md file, that could be rendered with any md renderer would be helpful and a great feature.

**Comment by RadhiFadlillah** [comment]
I'm sorry, but unfortunately I doubt I could do it :cry:.

The main reason is, the HTML output of `go-readability` (and `Readability.js`) is only readable, but not by any means is clean. For example, sometimes the readable content could be nested deep inside div, or it could be separated in several `divs`. This is why it's hard to parse and convert to markdown.

The best I can do is make it print the text content instead of HTML, which I might do later.

**Comment by Seirdy** [comment]
It's probably a better idea to just pipe the output of `go-readability` to another tool like clean-mark, `pandoc -f html -t markdown`, `w3m -T text/html -dump`, etc.
